test/WebTestCase.php

```diff
@@ -13,23 +16,47 @@
 
     final protected function logIn(Client $client)
     {
+        $client->request('GET', '/?open-sesame');
+
         $session = $client->getContainer()->get('session');
 
         $token = new OAuthToken(
             [
                 'access_token' => 'token',
                 'expires_in' => 3920,
                 'token_type' => 'Bearer',
+                'id' => 'jcarberry',
```

Comment:
is this a new standard field introduced by `profiles`?

Reply:
Yeah, it's the eLife ID. We haven't removed the ORCID extension to the token, so just extended it. We'd need to add a new endpoint that returns the eLife ID or user token or whatever.

Reply:
Related to #743